### ISSUE TITLE
Make core removal optional

### ISSUE THREAD

**Issue by Idhrendur** [Idhrendur/paradoxGameConverters#226]
Until some of the future work is completed, this option can remove colonial nations entirely.

See this forum post: https://forum.paradoxplaza.com/forum/index.php?threads/vic2-to-hoi4-converter-release-thread.948992/page-13#post-22424635

**Comment by derHochmeister** [comment]
It should, however, keep some special countries even if not existing. Notably, Canada, Australia, New Zealand, India, China, South Africa, Germany and Russia/Soviets.

**Comment by derHochmeister** [comment]
Partly referenced in https://github.com/Idhrendur/paradoxGameConverters/issues/250

**Comment by Idhrendur** [comment]
Yep, #250 would remove the need for this. And I should have quoted the original:

"I have a complaint/bug report for the newest version of the converter.

I recently converted my latest Vic2-game (which was itself converted from EU4) to Hoi4 and was quite upset when I noticed that most of the old EU4 colonial nations that were annexed and reduced to core during the EU4-Vic2-Conversion were now gone complete.

Releasing those was supposed to be my goal for this game!

I knew it was'nt like that in earlier versions of the converter.

While I applaud the effort to cut down the number of useless countries for HoI4, I would argue that this is not one of the situations were this is adequate.
When I defeat a colonial empire as a democratic or communist power, releasing the former colonies is an obvious thing to do.
Otherwise I only have the choice between letting them keep them or taking them for myself - neither of which is a particulary satisfying option in this scenario."

So this would be an option that would leave all those countries, as opposed to the current methodology which removes them.
